include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -209,6 +210,10 @@ table KUKAiiwaArmConfiguration {
   /// position, cartesian impedence, or joint impedence low level controller adjustments
   controlMode:EControlMode;
 
+  setCartImpedance:CartesianImpedenceControlMode;
```

Comment:
hmm, I wonder if the union is really best... I think you're right and these should just be available table entries like smartServoConfig and FRIConfig.